deploy/helm_charts/envs/bard.yaml

```diff
@@ -47,3 +47,6 @@ serviceGroups:
 
 svg:
   blocklistFile: ["dc/g/Uncategorized", "oecd/g/OECD"]
+
+cronTesting:
```

Comment:
Can you add 

```
nodejs:
  enabled: true
  replicas: 50
```
?

Now nodejs is not enabled in bard

Reply:
I'm not sure who's standing up the bard instance and whether or not they've completed the setup, because adding this would be part of that whole setup process right? so I was just going to add the testing changes so that once nodejs is up and running on bard, it can do the tests

Reply:
I see. With `cronTesting: enabled`, would the cron testing start to run every 4 hours?

Reply:
no, `cronTest: enabled` just writes values to the cronTesting configMap during deployment

also, when cron testing runs, it will check that there is a node server & if no node server, it will skip the nodejs query tests